examples/data-objects/inventory-app/tests/main.test.ts

```diff
@@ -21,4 +24,34 @@ describe.skip("Main", () => {
 	it("The page loads", async () => {
 		// await expect(page).toClick("button", { text: "Roll" });
 	});
+
+	it("works with container schema", async () => {
+		// await expect(page).toClick("button", { text: "Roll" });
```

Comment:
Is this no-op test desired? Neither this nor the test above actually validate anything (beyond that the page doesn't throw on load). Do we have anything tracking getting these tests working?

Reply:
oops, I added that when deciding to make a separate suite since this one is skipped. I'll remove it.

I can't run any of our jest tests locally (since they all give file watching errors) despite the entire test suite being skipped and the test bodies being commented out. Somone should care about that someday. I don't know if we are tracking the very bad state of many of our examples having no tests.

Reply:
Fixed (well I delated the extra test, we still have skipped commented out tests with broken IntelliSense which I can't run locally)